packages/app/src/sections/leaderboard/Leaderboard.tsx

```diff
@@ -195,15 +195,15 @@ const Leaderboard: FC<LeaderboardProps> = ({ compact, mobile }) => {
 						/>
 					) : searchAddress ? (
 						<AllTime
-							stats={stats.search}
+							stats={stats.search as any}
```

Comment:
Again would be great if we didn't have to cast to any

Reply:
Moving to the SDK helped fix this.